controllers/postgresqldatabase_controller_test.go

```diff
@@ -207,7 +207,7 @@ func TestPostgreSQLDatabase_Reconcile_hostCredentialsResourceReference(t *testin
 			Spec: lunarwayv1alpha1.PostgreSQLDatabaseSpec{
 				Name:            databaseName,
 				HostCredentials: hostCredentialsName,
-				Password: lunarwayv1alpha1.ResourceVar{
```

Comment:
Should we have a test without the password, or do you already have that covered in an earlier pr? I can't remember

Reply:
Yeah, it is [here](https://github.com/lunarway/postgresql-controller/blob/d237f3a58bd2b1e78a157e0efb0b58e9c657aa4e/controllers/postgresqldatabase_controller_test.go#L266). Unfortunately it does not catch the problem that is fixed in the current PR as the error is caught because this is not considered an error state for the controller.

To test it we would need a test setup that supported writing the status back to the postgresqldatabase resource.

Reply:
Yeah, that is maybe a bit too much for this pr. Lets keep it here, and test it in dev, the best way we can.